src/checkers/inference/InferenceAnnotatedTypeFactory.java

```diff
@@ -166,6 +166,14 @@ public CFTransfer createFlowTransferFunction(CFAbstractAnalysis<CFValue, CFStore
 
     @Override
     public TreeAnnotator createTreeAnnotator() {
+        if (realTypeFactory instanceof InferrableAnnotatedTypeFactory) {
+            TreeAnnotator realInferenceTreeAnnotator = ((InferrableAnnotatedTypeFactory) realTypeFactory)
+                    .getRealInferenceTreeAnnotator(this, realChecker,
+                            realTypeFactory, variableAnnotator, slotManager);
+            return new ListTreeAnnotator(new ImplicitsTreeAnnotator(this),
```

Comment:
Does this mean change the return type of getInferenceTreeAnnotator, and let it return ListTreeAnnotator rather than just a TreeAnnotator?